ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py

```diff
@@ -688,3 +689,48 @@ def test_torch_from_dlpack(
         fn_tree=fn_tree,
         on_device=on_device,
     )
+
+
+@st.composite
+def _get_dtype_buffer_count_offset(draw):
+    dtype, value = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+        )
+    )
+    value = np.array(value)
+    length = value.size
+    value = value.tobytes()
```

Comment:
Hey @V-G-spec, here's my inference after experimenting with both Numpy and its Ivy equivalent:

### Case 1: Numpy
On converting a numpy array to bytes, we get the following:

```
>>> np_array = np.array([1, 2])
>>> bytes = np_array.tobytes()
>>> bytes
b'\x01\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00' 
```

### Case 2: Ivy 
Whereas on converting an Ivy array of the same size, there's a difference in the number of bytes 

```
>>> ivy_arr = ivy.array([1, 2]) 
>>> bytes = ivy_arr.tobytes()
>>> bytes
b'\x01\x00\x00\x00\x02\x00\x00\x00'
```